core/src/banking_stage/unified_scheduler.rs

```diff
@@ -0,0 +1,94 @@
+//! This module contains any integration code to put the unified scheduler subsystem into the
+//! banking stage, starting from `BankingPacketBatch` ingestion from the sig verify stage and to
+//! `Task` submission to the unified scheduler.
+//!
+//! These preprocessing for task creation can be multi-threaded trivially. At the same time, the
+//! maximum cpu core utilization needs to be constrained among this processing and the actual task
+//! handling of unified scheduler. Thus, it's desired to share a single thread pool for the two
+//! kinds of work. Towards that end, the integration was implemented as a callback-style, which is
+//! invoked (via `select!` on `banking_packet_receiver`) at each of unified scheduler handler
+//! threads.
+//!
+//! Aside from some limited abstraction leakage to make `select!` work at the
+//! solana-unified-scheduler-pool crate, almost all of these preprocessing are intentionally
+//! encapsulated into this module, at the cost of dynamic dispatch per each BankingPacketBatch to
+//! retain the unified scheduler agnostic scheduler over scheduling mode (block verification vs
+//! block production) as much as possible.
+//!
+//! Lastly, what the callback closure in this module does is roughly as follows:
+//! 1. Translate the raw packet bytes into some structs
```

Comment:
support of the recently landed TransactionView is out of scope for this pr as well.